src/services/tweets.service.ts

```diff
@@ -24,15 +27,23 @@ export const getTweetById = (id: string) => {
   return getOneById(id);
 };
 
-export const createTweet = (
+export const createTweet = async (
   author: string,
   content: string,
   repliedTo?: string
 ) => {
-  return createOne(author, content, repliedTo);
+  const createResult = await createOne(author, content, repliedTo);
+
+  if (createResult) {
```

Comment:
what do you check here?
if error we did not come here
if object we had always true